java/src/jmri/jmrit/logixng/actions/ActionBundle_cs.properties

```diff
@@ -52,6 +52,9 @@ ActionEntryExit_DestinationPointsInUseVeto  = C\u00edlov\u00fd bod je pou\u017e\
 ActionListenOnBeans_Short           = Poslouchat na beanech
 ActionListenOnBeans_Long            = Poslouchat na beanech
 
+ActionListenOnBeansTable_Short      = Poslouchat na beanech - Tabulka
+ActionListenOnBeansTable_Long       = Poslouchat ka\u017ed\u00fd {0} v ka\u017ed\u00e9m {1} z {2} "{3}" v tabulce "{3}"
```

Comment:
Poslouchat ka\u017ed\u00fd {0} v ka\u017ed\u00e9m {1} z {2} "{3}" v tabulce "{3}"
should be
Poslouchat ka\u017ed\u00fd {0} v ka\u017ed\u00e9m {1} z {2} "{3}" v tabulce **"{4}"**

It's my fault. The error is in ActionBundle.properties as well, but fixed in PR #9867.